Gemfile

```diff
@@ -57,6 +57,7 @@ gem 'paperclip',                      "~>4.3.0"
 
 # Not vendored and not required
 gem "ancestry",                       "~>2.1.0",      :require => false
+gem "azure-armrest",                                  :path => "/Users/bronaghsorota/dev/azure-armrest"
```

Comment:
this path will only work on your machine.  You can either point at a version of the gem or, I think, a specific git reference.  cc @Fryguy